### ISSUE TITLE
Ensure calculations for weighting is true to the original algorithm

### ISSUE THREAD

**Issue by jakewilliami** [jakewilliami/FaceDetection.jl#5]
`print(weights)` at the line before [this line](https://github.com/Simon-Hohberg/Viola-Jones/blob/master/violajones/AdaBoost.py#L68) will print the array
```
[0.00020585 0.00020585 0.00020585 ... 0.00020585 0.00020585 0.00020585]
```
And `print(weights)` after said line will print arrays
```
[0.00020585 0.00020585 0.00020585 ... 0.00020585 0.00020585 0.00020585]
[0.00020585 0.00020585 0.00020585 ... 0.00020585 0.00020585 0.00020585]
[0.00013831 0.00013831 0.00013831 ... 0.00013831 0.00030636 0.00030636]
```
I need to replicate this in Julia.

**Comment by jakewilliami** [comment]
Distinctly *not* fixed.  I thought this was fixed because I rectified the `featureIndices` size, but this is still incorrect.  May relate to the result of #11.

**Comment by jakewilliami** [comment]
Closed by 6b35f6d5.
